src/modules/LanItems.rb

```diff
@@ -786,156 +786,6 @@ def GetItemModules(default_module)
       deep_copy(mods)
     end
 
-    # Searches map of known devices and decides if referenced lan item
-    # can be enslaved in a bond device
-    #
-    # @param [String] bondMaster    name of master device
-    # @param [Fixnum] itemId        index into LanItems::Items
-    # TODO: Check for valid configurations. E.g. bond device over vlan
-    # is nonsense and is not supported by netconfig.
-    # Also devices enslaved in a bridge should be excluded too.
-    def IsBondable(bondMaster, itemId)
```

Comment:
Yeah, dropping code is nice :)

Reply:
it was purpose of this PR. Keep what can be kept from LanItems and move it to new API to relax dependency on LanItems